server/utils/README.md

```diff
@@ -0,0 +1,7 @@
+# **API call to fetch 311 request data from the 311 data server for a given start date and end date.** 
+
+### The 311 request data from the [lacity.org](https://data.lacity.org/browse?q=MyLA311%20Service%20Request%20Data%20&sortBy=relevance) has 34 columns. The get_request_data_csv.py script can be run from the command line passing the arguments- start_date and end_date that lets you retreive the 311 request data from the [311 data server](https://dev-api.311-data.org/docs). The 311 server processes the data from the lacity.org. The data cleaning procedure is mentioned [here](https://github.com/hackforla/311-data/blob/dev/docs/data_loading.md). The dataframe that is returned can be saved as a csv file. A preview of the data_final dataframe is printed in the command line. 
```

Comment:
nit: to be clear, we are not returning a dataframe. We don't even need to mention that a dataframe is being produced, since it's an intermediate result. We can just say "The result is written to a csv file". We should also explicitly say what the path to the output csv file is.